src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/DataSetVersionMappingControllerTests.cs

```diff
@@ -504,16 +504,125 @@ public async Task Success_AddedLevel_MinorUpdate()
                                     .WithNoMapping())
                             .AddCandidate(
                                 targetKey: "target-location-1-key",
-                                candidate: DataFixture.DefaultMappableLocationOption()))
-                    // Country level has been added and only has candidates.
+                                candidate: DataFixture.DefaultMappableLocationOption())))
+                // There are deleted indicators that cannot be mapped.
+                .WithHasDeletedIndicators(true);
+
+            await TestApp.AddTestData<PublicDataDbContext>(context => context.DataSetVersionMappings.Add(mapping));
+
+            List<LocationMappingUpdateRequest> updates =
+            [
+                new()
+                {
+                    Level = GeographicLevel.LocalAuthority,
+                    SourceKey = "source-location-1-key",
+                    Type = MappingType.ManualMapped,
+                    CandidateKey = "target-location-1-key"
+                }
+            ];
+
+            var response = await ApplyBatchLocationMappingUpdates(
+                nextDataSetVersionId: nextDataSetVersion.Id,
+                updates: updates);
+
+            response.AssertOk<BatchLocationMappingUpdatesResponseViewModel>();
+
+            var updatedMapping = await TestApp.GetDbContext<PublicDataDbContext>()
+                .DataSetVersionMappings
+                .Include(m => m.TargetDataSetVersion)
+                .SingleAsync(m => m.TargetDataSetVersionId == nextDataSetVersion.Id);
+
+            Assert.True(updatedMapping.LocationMappingsComplete);
+
+            // This update completes the mapping and would normally be a minor version
+            // update, but the deleted indicators mean this is still a major version update.
+            await AssertCorrectDataSetVersionNumbers(updatedMapping, "2.0.0");
+        }
+
+        [Fact]
+        public async Task Success_MappedLocation_HasDeletedGeographicLevels_MajorUpdate()
```

Comment:
So what is the difference between `Success_MappedLocation_HasDeletedLocationLevels_MajorUpdate` and `Success_MappedLocation_HasDeletedGeographicLevels_MajorUpdate`?

Reply:
Think this relates to my earlier reply: https://github.com/dfe-analytical-services/explore-education-statistics/pull/5332#discussion_r1799207660